### DISCUSSION TITLE
[Carousel] Need advice to add a counter

### DISCUSSION THREAD

**Discussion by bckthomas** [fancyapps/ui#83] [Q&A] [answered]
Hi
I would like to add a counter like the screenshot below images. 
Can you give me a clue to code this feature please.

Thanks

<img width="250" alt="image" src="https://user-images.githubusercontent.com/57676204/130401401-0b3eadda-ea09-4f1f-9ae1-c0fe56410855.png">

**Comment by bckthomas** [comment]
Update : 
The counter is ready, but how can I update the current slide ? I don't find an event to execute code after slide event.
Can you help me ?

**Comment by fancyapps** [comment, marked as answer]
Hi,

See this demo - https://fancyapps.com/playground/FS

```JS
const myCarousel = new Carousel(document.querySelector("#myCarousel"), {
  Dots: false,
  on: {
    init: (carousel) => {
      carousel.$index = carousel.$container.querySelector(".carousel_index");
      carousel.$count = carousel.$container.querySelector(".carousel_count");
    },
    refresh: (carousel) => {
      if (carousel.$count) {
        carousel.$count.innerHTML = carousel.pages.length;
      }
    },
    change: (carousel) => {
      if (carousel.$index) {
        carousel.$index.innerHTML = carousel.page + 1;
      }
    },
  },
});
```

**Reply by bckthomas** [reply]
You're my here ! Thanks
